src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -39,6 +39,7 @@
 
 import git
 from reuse import _annotate, _util, lint, project
+from reuse.vcs import VCSStrategyGit
 
 DEFAULT_TEMPLATE = "ansys"
```

Comment:
Which other strategies are supported and would it make sense to accept on input these other strategies rather than default to VCSStrategyGit? I'm thinking about those users that benefit from other IDEs mostly.

Reply:
`reuse` provides VCSStrategy for Git, Mercurial, [Jujutsu](https://github.com/martinvonz/jj) and Pijul (https://nest.pijul.com/pijul/pijul)
AFAIK we are mainly using Git

Reply:
Okay yeah - let's leave it then. No clue about the last two 😄 and even Mercurial I rarely see it